airgun/entities/contentview.py

```diff
@@ -115,6 +140,65 @@ def promote(self, entity_name, version_name, lce_name):
         view.flash.dismiss()
         return view.versions.table.row(version=version_name).read()
 
+    def read_version(self, entity_name, version_name):
+        """Read content view version values"""
+        view = self.navigate_to(
+            self, 'Version',
+            entity_name=entity_name,
+            version_name=version_name,
+        )
+        return view.read()
+
+    def search_version(self, entity_name, query):
+        """Search for content view version"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.versions.search(query)
+
+    def search_version_package(
+            self, entity_name, version_name, query, repo=None):
+        """Search for a package inside content view version
+
+        :param str optional repo: repository name to filter by
```

Comment:
Think if we make a description for one param, we have to make for all of them, or add that param to desc.
```
Search for a package inside content view version, optionally we can indicate 
a repository to search by.
```
what do you think?